packages/api/tests/endpoints/test-execution-status.js

```diff
@@ -363,48 +368,51 @@ test('CUMULUS-912 GET with an invalid access token returns an unauthorized respo
 
 test.todo('CUMULUS-912 GET with an unauthorized user returns an unauthorized response');
 
-test('returns ARNs for execution and state machine', async (t) => {
+test.serial('returns ARNs for execution and state machine', async (t) => {
```

Comment:
@jennyhliu  not relevant for this PR, just out of curiosity, but in https://github.com/nasa/cumulus/pull/3123 was there a reason this was made serial?

Reply:
Because retrieving a execution status creates a s3 file, and we don't want to have multiple files created concurrently for the same execution